ParamConverter/DataTransferObjectParamConverter.php

```diff
@@ -140,12 +143,14 @@ private function autoConfigureProperty(
         Request $request,
         array $paramConfiguration,
         string $prefix,
+        ?string $dtoName,
         PropertyConfigurationExtractor $propertyConfigurationModel
     ): array {
-        $content = $request->request->all()[$propertyConfigurationModel->getName()] ?? null;
+        $name = $propertyConfigurationModel->getName();
+        $content = $dtoName === null ? $request->request->get($name) : ($request->attributes->get($dtoName)[$name] ?? null);
```

Comment:
Same thing as before : simplify(or documents)/add parenthesis

Reply:
I'd suggest this:

```suggestion
        $content = $dtoName
            ? ($request->attributes->get($dtoName)[$name] ?? null)
            : $request->request->get($name)
        ;
```